DESIGN.md

```diff
@@ -134,7 +134,7 @@ All commands to the shell itself will be prefixed with a backslash (\), e.g:
 * `\env`
   * Prints out your current environment settings including where they were set from.
 
-### Line terminators <a href="line-terminators"/>
+### Line terminators
```

Comment:
Currently it will render the whole section as hyperlink incorrectly https://github.com/awslabs/amazon-qldb-shell/blob/rust/DESIGN.md#line-terminators-